push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/holder/AllBatchesLoaded.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * JBoss, Home of Professional Open Source
+ * Copyright Red Hat, Inc., and individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.aerogear.unifiedpush.message.holder;
+
+import java.io.Serializable;
+
+/**
+ * Event fired when all batches for given variant are loaded and dispatched for further processing
+ */
```

Comment:
not exactly, when the loading process is done, it does not mean Senders processed those batches.

See `VariantCompleted` which distinguishes the end of whole sending process.